### DISCUSSION TITLE
help getting RHEL 8.2 running under WSL2 and registered

### DISCUSSION THREAD

**Discussion by devLinuxNC** [microsoft/WSL#5926] [General]
Where would I go to get help getting an instance of RHEL 8.2 running under WSL2?

I have posted on StackOverflow, but had no takers (after months).

I don't completely understand all the MS docs.  I don't use VSCode, so that is not an option.

I would like to get something running without a dependency on VSCode.

I'm not sure what I would download, or where to get it.  I don't want the full blown RHEL 8.2 distribution with all the GUI stuff, just a shell, and have it registered to get updates from AppStream.

If anyone has done this, it would be a great help to get instructions or the recipe followed.

thank you,

**Comment by Biswa96** [comment]
> Where would I go to get help getting an instance of RHEL 8.2 running under WSL2?

Here https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux

> I have posted on StackOverflow, but had no takers (after months).

Maybe because it's commercial/enterprise software.

> I would like to get something running without a dependency on VSCode.

Neither WSL nor GNU/Linux systems depend on VSCode.

> I'm not sure what I would download, or where to get it.

Here https://developers.redhat.com/products/rhel/download. Requires account.

tl;dr: If you want to live in RedHat ecosystem just use Fedora or CentOS. Distributions do not matter.

**Comment by WSLUser** [comment]
You can get Pengwin Enterprise from the MS Store. If you have a Red Hat developer account, you can grab RHEL for free. The company behind Pengwin Enterprise, Whitewater Foundry, has a guide to migrate from Scientific Linux to RHEL. I also have a script to migrate Scientific Linux 7 to CentOS 7, if you are ok with not having the normally paid for support that comes with RHEL. https://github.com/WSLUser/ScientificLinuxtoCentOSTransition

**Reply by WSLUser** [reply]
Alternatively, just grab a docker image of CentOS 8 and import it as a WSL2 instance. Migrating to RHEL from CentOS should be simple if you have the license to use RHEL.
